File: Beecrowd/1150 Ultrapassando Z/main.py
cont = 0 
soma = 0

X = int(input())

# uma forma de fazer do while
while True:    
  Z = int(input())
  if Z > X:
    break

i = X
while soma <= Z:
  soma += i
  cont += 1
  i += 1
    
print(cont)

File: Beecrowd/1150 Ultrapassando Z/main.c
#include <stdio.h>

int main(){
  int X, Z, i, cont = 0, soma = 0;
  
    scanf("%d", &X);
    
    do{
      scanf("%d", &Z);
    }while(Z <= X);

    i = X;
    while(soma <= Z){
      soma += i;
      cont++;
      i++;
    }
    
    printf("%d\n", cont);

  return 0;
}